go/arrow/arrio/arrio_test.go

```diff
@@ -98,15 +98,15 @@ func TestCopy(t *testing.T) {
 							if err != nil {
 								t.Fatal(err)
 							}
-							defer f.Close()
 							defer os.Remove(f.Name())
+							defer f.Close()
```

Comment:
hum... perhaps we should just create a per-sub-test tmp directory and `defer os.RemoveAll(tmpdir)` instead?
(and loose the `defer os.Remove(file.Name())` calls)